tests/requirements-tools.txt

```diff
@@ -5,10 +5,10 @@ git+https://github.com/pytest-dev/execnet.git
 pytest >= 2.7.3  # Testing framework.
 pytest-xdist     # Plugin allowing running tests in parallel.
 pytest-timeout   # Plugin to abort hanging tests.
-pytest-catchlog  # Capture log output separately from stdout/stderr
```

Comment:
Why? I was distracted by the logs written out, too, but are the other reasons? And the reason should be stated in teh commit message.

Reply:
IC:
>  pytest-catchlog plugin has been merged into the core, please remove it from your requirements.

Reply:
This is in fact what makes this PR possible. `pytest-catchlog` was *one* of two problems that impaired parallel testing. It's been merged into pytest core.